docs/sections/getting_started/basics.md

```diff
@@ -305,30 +291,35 @@ For this purpose, AiiDA provides the `CalcJob` process class.
 
 To see all calculations available from the AiiDA packages installed in your environment you can use the `verdi plugin` command:
 
-:::{margin}
-If you just run `verdi plugin list`, you will get a list of all possible groups of plugins.
-:::
-
 ```{code-block} console
-
 $ verdi plugin list aiida.calculations
+```
+
+This will show you a long list of _entry points_: strings that are used to identify each plugin within AiiDA.
+In this list you should be able to see the `arithmetic.add` entry point, which identifies the calculation job we want to run:
+
+```{code-block} console
 Registered entry points for aiida.calculations:
 (...)
 * arithmetic.add
 (...)
 
 Info: Pass the entry point as an argument to display detailed information
-
 ```
 
-This will show you a long list of _entry points_: strings that are used to identify each plugin within AiiDA.
-In this list you should be able to see the `arithmetic.add` entry point, which identifies the calculation job we want to run.
+:::{note}
+If you just run `verdi plugin list`, you will get a list of all possible plugin _groups_.
+:::
```

Comment:
Any reason in particular why you set this as a note instead of a margin? I'm not particularly against it but just to understand. I originally set it as margin so as to lower its importance and so it wouldn't be in the middle of the hands on text.

Reply:
This hasn't been hashed out 100%, but so far I'm using margins for:

1. Further reading
2. Downloading code snippets
3. Quick reminders (only "Don't forget the PK" so far)

This seemed like a regular note to me. We'll have to decide on proper guidelines for how to use the margins at some point though.